lib/clientsidescripts.js

```diff
@@ -72,7 +72,10 @@ functions.findBindings = function(binding, exactMatch, using, rootSelector) {
     if (dataBinding) {
       var bindingName = dataBinding.exp || dataBinding[0].exp || dataBinding;
       if (exactMatch) {
-        var matcher = new RegExp('({|\\s|^|\\|)' + binding + '(}|\\s|$|\\|)');
+        var matcher = new RegExp('({|\\s|^|\\|)' +
+            // See http://stackoverflow.com/q/3561711
```

Comment:
Single line comments inside client side scripts can screw things up - see the note at the top of this file. Make this `/* */` style.